### DISCUSSION TITLE
Should we release 0.69.10?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#63] [Patches]
## Released, please refer to #68 

---

Current Release: [0.69.9](https://github.com/facebook/react-native/releases/tag/v0.69.9)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. 

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
once we tested them in 0.72's next RC (RC1) - let's cherry pick @NickGerleman's fixes for TextInput (EditText) (in order):
* https://github.com/facebook/react-native/commit/1743dd7ab40998c4d3491e3b2c56c531daf5dc47
* https://github.com/facebook/react-native/commit/b9e2627d1ce64e8e293e90f743bf13a515db3b4d
* https://github.com/facebook/react-native/commit/cc0ba57ea42d876155b2fd7d9ee78604ff8aa57a
* https://github.com/facebook/react-native/commit/8c9c8ba5adb59f7f891a5307a0bce7200dd3ac7d
* https://github.com/facebook/react-native/commit/0869ea29db6a4ca20b9043d592a2233ae1a0e7a2
* https://github.com/facebook/react-native/commit/5791cf1f7b43aed1d98cad7bcc272d97ab659111
* https://github.com/facebook/react-native/commit/104cb7f81ecbb81841070078c55ddf5d8ea9d24b
* https://github.com/facebook/react-native/commit/b384bb613bf533aebf3271ba335c61946fcd3303
* https://github.com/facebook/react-native/commit/92b898149956a301a44f99019f5c7500335c5553

**Reply by NickGerleman** [reply]
Versions before 0.72 will need this change cherry-picked first https://github.com/facebook/react-native/commit/8a0fe30591e21b90a3481c1ef3eeadd4b592f3ed

**Reply by kelset** [reply]
PR: https://github.com/facebook/react-native/pull/36876

**Comment by pierroo** [comment]
To the question "should we release 0.69.10"
=> By all means, please do!
